scheduler-server/sql/scheduler/applied_migrations.sql

```diff
@@ -2,4 +2,6 @@
 This file denotes which migrations occur "before" this version of the schema.
 */
 
-call migrations.mark_migration_applied('0');
+call mark_migration_applied('0');
```

Comment:
I think you need to specify the schema here, as I believe this runs by default in the `public` schema.

Reply:
Thank you, done :+1: